geniuspaste/data/tinypaste.com.conf

```diff
@@ -0,0 +1,20 @@
+# FIXME: outdated, this leads to a 404 on http://pasted.co/
```

Comment:
Yes, this doens't work.  It didn't before my changes either, I guess the service changed since it was implemented.
I couldn't find how to use it, and their link to a supposed API documentation leads to a 404.

I'd delete this pastebin configuration altogether if I was the only caller, but for now I kept it in the hope maybe someone knows how to fix it.